spec/0.0.2/examples/template/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM goern/atomicapp-centos7 
```

Comment:
I think we should follow OpenShift's lead and put the base image name in the atomicapp name... atomicapp-rhel7 atomicapp-centos7 ...